packages/@ourworldindata/grapher/src/closeButton/CloseButton.scss

```diff
@@ -0,0 +1,45 @@
+.close-button {
+    $light-stroke: #e7e7e7;
+
+    $active-fill: #dbe5f0;
+    $hover-fill: #f2f2f2;
+
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    position: relative;
+    height: 32px;
+    width: 32px;
+    padding: 7px;
+
+    color: $light-text;
+    background: white;
+    border: 1px solid $light-stroke !important;
+    border-radius: 50%;
+
+    &:hover {
+        background: $hover-fill;
+        cursor: pointer;
+
+        &:not(.active) {
+            color: $dark-text;
+        }
+    }
+
+    &.active,
+    &:active {
+        background: $active-fill;
+        border: 1px solid $active-fill;
+    }
+
+    &.active {
+        cursor: default;
+        color: $active-text;
+    }
```

Comment:
I think the `.active` class name is never used anywhere - could you check and see if it's still relevant?

Reply:
Removing it...